docs/tutorials/zq-primer.md

```diff
@@ -249,13 +249,13 @@ written as `[int64]`.
 `v9` is a "set of strings", which is written like an array but with the
 enclosing syntax `|[` and `]|`.
 
-`v10` is a "map" type, which in other languages are often called a "table"
+`v10` is a "map" type, which in other languages is often called a "table"
 or a "dictionary".  In Zed, a value of any type can be used for the key or the
 value though all of the keys and all of the values must have the same type.
 
 Finally, `v11` is a Zed "record", which is similar to a JSON "object", but the
 keys are called "fields", the order of the fields is significant and
-is always preserved, and the field values all have a uniform type.
+is always preserved.
```

Comment:
The removed text "...and the field values all have a uniform type" didn't make sense to me, so I checked with @mccanne on Slack and confirmed that he doesn't remember what he meant by that, so I've proposed removing it. Maybe someone can recall why it was there and suggest something to put in its place that's clearer.